flex/engines/graph_db/app/server_app.cc

```diff
@@ -22,7 +22,7 @@ static uint32_t get_vertex_vid(const gs::ReadTransaction& txn, uint8_t label,
   uint32_t vid = std::numeric_limits<uint32_t>::max();
   auto vit = txn.GetVertexIterator(label);
   for (; vit.IsValid(); vit.Next()) {
-    if (vit.GetId() == id) {
+    if (vit.GetId().AsInt64() == id) {
```

Comment:
If the result of `vit.GetId()` must be `int64()`, do the check first?

Reply:
Checked inside the AsXX() function